app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -254,6 +252,10 @@ def display_phone_to_deliver_to
     end
   end
 
+  def decorated_user
+    current_user.decorate
+  end
```

Comment:
It's not clear to me what problem this PR is solving. Why remove the `decorated_user` method from application controller, and then define it here to be able to use this helper method, but not everywhere else? It seems inconsistent to call `current_user.decorate` directly in all controllers except this one.

What if we change the way we decorate the current user? Wouldn't it be easier to change a single `decorated_user` method rather than all instances of `current_user.decorate`?